tests/system/common.py

```diff
@@ -83,3 +85,43 @@ def wait_for_mesos_endpoint(timeout_sec=timedelta(minutes=5).total_seconds()):
     it returns false"""
 
     return shakedown.time_wait(lambda: shakedown.mesos_available_predicate(), timeout_seconds=timeout_sec)
+
+
+def metronome_available_predicate():
+    url = metronome_api_url()
+    try:
+        response = http.get(url)
+        return response.status_code == 200
+    except Exception as e:
+        return False
+
+
+def metronome_api_url():
+    return shakedown.dcos_url_path("/service/metronome/v1/jobs")
+
+
+def assert_job_run(client, job_id, runs_number=1, tasks_number=1):
+    job_runs_count = len(client.get_runs(job_id))
```

Comment:
Now that I'm thinking about it - does `get_runs` give you active runs? scheduled? past? All 3 of them?

Reply:
active

Reply:
Shouldn't we test for all 3 of them? In case the job run already finished (or scheduler?). Does it make sense?

Reply:
yes, makes sense, I'll fix it

Reply:
I found out there is probably no way how to get finished runs https://dcos.github.io/metronome/docs/generated/api.html# this is the closest thing and I am not sure it's worth it https://avarkocko-elasticl-c24kttf2fnwz-1817360483.us-west-2.elb.amazonaws.com/service/metronome/v1/jobs/test?embed=history